SilMock/DataStore/Sqlite/SqliteUtils.php

```diff
@@ -278,33 +282,33 @@ public function createDbStructureAsNecessary()
      * Run the given SQL statement as a PDO prepared statement, using the given
      * array of data.
      *
-     * @param string $sql The SQL statement. Example: "SELECT * FROM table WHERE
-     * id = :id"
-     * @param array $data (Optional:) An associative array where the keys are
-     * the placeholders in the SQL statement. Example: array(':id' => 1).
+     * @param string $sql The SQL statement.
```

Comment:
Okay, I wasn't completely consistent, but the indented line below helps make the semantic break visible too.